src/shared/config/frontend.config.ts

```diff
@@ -1,11 +1,29 @@
-import { IsBoolean, IsInt, IsNotEmpty, IsPositive, IsString, Max, Min } from 'class-validator';
+import {
+    IsBoolean,
+    IsInt,
+    IsNotEmpty,
+    IsOptional,
+    IsPositive,
+    IsString,
+    Max,
+    Min,
+    isBoolean,
+    isInt,
+    isString,
+} from 'class-validator';
+
+import { OneOf } from '../util/one-of.validator.decorator.js';
 
 export class FrontendConfig {
     @IsInt()
     @Min(1024)
     @Max(10000)
     public readonly PORT!: number;
 
+    @IsOptional()
+    @OneOf(isString, isInt, isBoolean)
```

Comment:
I am unsure if passing predicates or passing types is better here and could not find any best practices so far. We should discuss this @kristoff-kiefer